### PR TITLE
Switching port arguments for the CLI to ENV variables

### PR DESCRIPTION
**Pull request by mcharawi**
@kevgo Addresses Originate/exosphere-sdk#69

### CODE REVIEW

File: features/support/cli-world.ls [hunk 1]
```diff
@@ -17,9 +17,11 @@ require! {
 CliWorld = !->
 
   @create-exocom-instance = (ports, done) ->
+    process.env.EXOCOM_HTTP_PORT = ports.http-port
```
Comment (kevgo) [inline features/support/cli-world.ls]:
```ls
env =
  EXOCOM_HTTP_PORT: ports.http-port
  EXOCOM_ZMQ_PORT: ports.zmq-port

File: features/support/cli-world.ls [hunk 2]
```diff
@@ -17,9 +17,11 @@ require! {
...
+    process.env.EXOCOM_ZMQ_PORT = ports.zmq-port
     @exocom-http-port = ports.http-port
     @exocom-zmq-port = ports.zmq-port
-    @process = new ObservableProcess "bin/exocom --zmq-port #{@exocom-zmq-port} --http-port #{@exocom-http-port}", console: my-console
+    @process = new ObservableProcess "bin/exocom", console: my-console
```
Comment (kevgo) [inline features/support/cli-world.ls]:
```
@process = new ObservableProcess "bin/exocom", console: my-console, env: env

Comment (kevgo) [inline features/support/cli-world.ls]: https://github.com/originate/observable-process#set-environment-variables-in-the-subshell

File: features/support/env.ls [hunk 3]
```diff
@@ -10,3 +10,5 @@ module.exports = ->
     @process?.kill!
     for name, mock of @service-mocks or {}
       mock.close!
+    delete process.env.EXOCOM_ZMQ_PORT
```
Comment (kevgo) [inline features/support/env.ls]: This can go away

File: src/cli.ls [hunk 4]
```diff
@@ -34,11 +34,13 @@ on-error = (err) ->
...
 
 run = ->
+  process.env.EXOCOM_ZMQ_PORT = 4100 unless process.env.EXOCOM_ZMQ_PORT
+  process.env.EXOCOM_HTTP_PORT = 4101 unless process.env.EXOCOM_HTTP_PORT
   exocom = new ExoCom!
     ..on 'zmq-bound', on-zmq-bound
     ..on 'http-bound', on-http-bound
     ..on 'error', on-error
-    ..listen zmq-port: (options['--zmq-port'] or 4100), http-port: (options['--http-port'] or 4101)
+    ..listen zmq-port: +process.env.EXOCOM_ZMQ_PORT, http-port: +process.env.EXOCOM_HTTP_PORT
```
Comment (kevgo) [inline src/cli.ls]: No point in assigning these variables to process.env when you simply read them here again.

**Comment by mcharawi** [comment]
@kevgo comments addressed

**Comment by kevgo** [comment]
⛵

**Pull request closed without merging**